src/Drips.sol

```diff
@@ -501,9 +501,10 @@ library Drips {
                     newDefaultEnd
                 );
                 _setDeltaRange(state.amtDeltas, cycleSecs, start, end, newRecv.amtPerSec);
-                // The receiver may have never been used, initialize it
-                if (state.nextCollectedCycle == 0) {
-                    state.nextCollectedCycle = _currTimestamp() / cycleSecs + 1;
+                // Ensure that the receiver collects the updated cycles
+                uint32 startCycle = start / cycleSecs + 1;
```

Comment:
I am wondering if we should back this `+1` operation in some helper method?

Reply:
Fixed in the other PR